sequelizeDatabase/modelSetup.js

```diff
@@ -0,0 +1,135 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('localDatabaseSean','root','ConnectedNetwork1',{
```

Comment:
`localDatabaseSean` 🤣 
I like it. We should probably make an issue to extract this to a `.env` file